indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -68,6 +92,35 @@
         // we can't iterate over this in-place, so do this instead
         while (form.elements.provisions.selectedOptions.length > 0) form.elements.provisions.selectedOptions[0].selected = false;
       }
+    },
+
+    handleCheckboxesChange(e) {
+      function toggleSelectionByCheckSelected(currentCheckbox) {
+        const parentNodeListItem = currentCheckbox.closest("li");
```

Comment:
Once we've got the LI that this checkbox belongs to, can't we just find all checkboxes beneath it and toggle their state? Something like:

```js
for (const checkbox of currentCheckbox.closest('li').querySelectorAll('ul input[type=checkbox]')) {
  checkbox.checked = currentCheckbox.checked;
}
```

Reply:
done